examples/demos/cloud_export_demo/main.go

```diff
@@ -4,9 +4,10 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/kentik/community_sdk_golang/apiv6/examples/demos"
-	"github.com/kentik/community_sdk_golang/apiv6/kentikapi"
-	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/cloudexport"
+	cloudexport2 "github.com/kentik/community_sdk_golang/kentikapi/cloudexport"
```

Comment:
Why is it called "caloudexport2"...?

Reply:
Since there were to imports ending in cloudexport GoLand must have created an alias for the added cloudexport. I didn't notice it at the time. Similarly with synthetics.